### DISCUSSION TITLE
What is .obm file?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#203] [Q&A] [answered]
I noticed with my own build of Objeck that whenever I compiled a source file, an obm file was also created. This doesn't happen with the official release. What is the purpose of that file?

**Comment by objeck** [comment, marked as answer]
What toolchain? What is the compile command? The .obm file is a debug file that contains "human readable" bytecode.

**Reply by ghost** [reply]
I build my own version of Objeck from your latest code using MSYS2 UCRT64.

**Reply by ghost** [reply]
Compile command is just `obc -src`
